counterpoint/lib/mark.ts

```diff
@@ -678,6 +677,15 @@ export class Mark<AttributeSet extends AttributeSetBase = MarkAttributes>
     this._reverseAdjacency.delete(sourceMark);
     return this;
   }
+  /**
+   * Sets transform of an attribute to a new function, can be applied to an Attribute,
+   * a Mark, or a MarkRenderGroup
+   * @param attrToModify attribute to modify
+   * @param newFunc new function to set transform to
+   */
+  setTransform(attrToModify: string, newFunc : (raw: any, computeArg : any) => any) : void {
+    this.attributes[attrToModify].transform = newFunc;
```

Comment:
Let's directly call the `Attribute.setTransform` method that you wrote in the `Mark` and `MarkRenderGroup` methods, so that if the internal `Attribute` implementation needs to change, these ones don't